pythran/config.py

```diff
@@ -13,6 +13,11 @@
 logger = logging.getLogger('pythran')
 
 
+def get_include():
+    # using / as separator as advised in the distutils doc
+    return (os.path.dirname(os.path.dirname(__file__)) or '.') + '/pythran'
```

Comment:
Ah, the double `dirname` just strips off one directory level. And then you're adding it back with `/pythran`.

```python
>>> os.path.dirname(os.path.dirname('/Users/rgommers/Code/pythran/config.py'))
'/Users/rgommers/Code'
>>> os.path.dirname('/Users/rgommers/Code/pythran/config.py')
'/Users/rgommers/Code/pythran'
```

Reply:
exactly :-) and the `or` part is just here for the case where we have a relative path  (but when does it happen?)